### PR TITLE
Feature/5482 login authorization

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Actions/Collections/PageActionCollection.cs [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
+
 namespace OrderFormAcceptanceTests.Actions.Collections
 {
-    public sealed class PageActionCollection
-    {
-    }
+	public sealed class PageActionCollection
+	{
+		public Authentication Authentication { get; set; }
+		public Pages.Homepage Homepage { get; set; }
```
Comment (talawson05) [inline src/OrderFormAcceptanceTests.Actions/Collections/PageActionCollection.cs:9]: Pages.Homepage ?

File: src/OrderFormAcceptanceTests.Steps/Steps/Login_Authorisation.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        [Then(@"will be taken to the Order Form Feature")]
+        public void ThenWillBeTakenToTheOrderFormFeature()
+        {
+            Test.Driver.Url.Should().Contain("/order/");
+        }
+        
+        [Then(@"the User will be informed they cannot access that feature")]
+        public void ThenTheUserWillBeInformedTheyCannotAccessThatFeature()
+        {
+            Test.Pages.OrderForm.ErrorTitle().Should().BeEquivalentTo("You're not authorised to view this page");
```
Comment (talawson05) [inline src/OrderFormAcceptanceTests.Steps/Steps/Login_Authorisation.cs:66]: happy with this much content in the test? How about contains "not authorised"?

File: src/OrderFormAcceptanceTests.Steps/Utils/EnvironmentVariables.cs [hunk 3]
```diff
@@ -67,6 +68,30 @@ private static string JsonConfigValues(string section, string defaultValue)
...
         }
 
+        internal static User User(UserType userType)
+        {
+            string userTypeString = string.Empty;
+            string defaultUserName = string.Empty;
+
+            switch (userType)
+            {
+                case UserType.Authority:
```
Comment (talawson05) [inline src/OrderFormAcceptanceTests.Steps/Utils/EnvironmentVariables.cs:78]: is authority right in this context?

Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Steps/Utils/EnvironmentVariables.cs:78]: It was either authority or NonBuyer, I thought we would be able to use authority for future tests

**Pull request merged**